app/policies/service_provider_mfa_policy.rb

```diff
@@ -22,13 +22,18 @@ def user_needs_sp_auth_method_verification?
     # If the user needs to setup a new MFA method, return false so they go to
     # setup instead of verification
     return false if user_needs_sp_auth_method_setup?
+    return false if !piv_cac_required? && !phishing_resistant_required?
```

Comment:
To do: The existing coverage of the class didn't require this line, but it was definitely necessary to ensure that the method returns `false` when no restrictions are imposed.

Reply:
> To do: The existing coverage of the class didn't require this line, but it was definitely necessary to ensure that the method returns `false` when no restrictions are imposed.

Expanded coverage in 049a8eb.